pants.ci.toml

```diff
@@ -3,12 +3,5 @@ dynamic_ui = false
 # Ensure colors are used (if your CI provider supports it). 
 colors = true
 
-[python]
-# Limit the maximum number of concurrent jobs used to resolve third
-# party dependencies. The max level of parallelism will be
-# `process_execution_local_parallelism x resolver_jobs`, but 
-# often you will have one resolve process at a time.
-resolver_jobs = 2
-
```

Comment:
As of 2.10, pants sets the number of concurrent jobs automatically based on the environment: https://www.pantsbuild.org/v2.10/docs/reference-python#section-resolver-jobs